packages/client/src/v2-events/features/events/actions/print-certificate/Review.tsx

```diff
@@ -179,15 +230,22 @@ export function Review() {
     ))
 
     if (confirmed) {
-      handleCertify?.()
-      actions.printCertificate.mutate({
-        eventId: fullEvent.id,
-        data: form,
-        transactionId: uuid(),
-        type: ActionType.PRINT_CERTIFICATE
-      })
-      clear()
-      navigate(ROUTES.V2.EVENTS.OVERVIEW.buildPath({ eventId }))
+      try {
+        const response = await onlineActions.printCertificate.mutateAsync({
+          eventId: fullEvent.id,
+          data: { ...formValues, templateId },
+          transactionId: uuid(),
+          type: ActionType.PRINT_CERTIFICATE
+        })
+        if (response) {
+          handleCertify?.()
+          navigate(ROUTES.V2.EVENTS.OVERVIEW.buildPath({ eventId }))
+        }
+      } catch (error) {
+        // TODO: add notification alert
```

Comment:
Could you please add a ticket so we remember to do this?

Reply:
okay